lib/git/environment.go

```diff
@@ -0,0 +1,15 @@
+package git
+
+import "github.com/Originate/git-town/lib/util"
+
+// EnsureIsRepository asserts that the current directory is in a repository
+func EnsureIsRepository() {
+	util.Ensure(isRepository(), "This is not a git repository.")
+}
+
+// Helpers
+
+func isRepository() bool {
```

Comment:
Nice, Hub does it in a similar way: https://github.com/github/hub/blob/master/git/git.go#L297-L301